timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/LocalPaxosComponents.java

```diff
@@ -96,8 +98,29 @@ public static LocalPaxosComponents createWithBlockingMigration(
             DataSource sqliteDataSource,
             UUID leaderUuid,
             boolean canCreateNewClients) {
-        LocalPaxosComponents components = new LocalPaxosComponents(metrics, paxosUseCase, legacyLogDirectory,
-                sqliteDataSource, leaderUuid, canCreateNewClients);
+        return createWithBlockingMigrationWithVersion(metrics,
+                paxosUseCase,
+                legacyLogDirectory,
+                sqliteDataSource,
+                leaderUuid,
+                canCreateNewClients,
+                Optional.empty());
+    }
+
+    public static LocalPaxosComponents createWithBlockingMigrationWithVersion(TimelockPaxosMetrics metrics,
```

Comment:
I think as with the other class, it's probably fine to overload this - no need to say `withVersion`